src/app/pages/server-details/server-details.component.ts

```diff
@@ -1208,7 +1243,92 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
 
       return undefined;
     }
+  }
+
+  public generateStaticWebChart(chartConf: any) {
+
+    const labelsField = chartConf.labelsField;
+    const scaleField = chartConf.scaleField;
+    const dataSet = this.benchmarksByCategory?.find(x => x.benchmark_id === chartConf.benchmark);
 
+    console.log(dataSet);
+
+    let charData: any;
+
+    if(dataSet && dataSet.benchmarks?.length) {
+      let labels: any[] = [];
+      let scales: number[] = [];
+      dataSet.benchmarks.forEach((item: any) => {
+        if(item.config[labelsField] && labels.indexOf(item.config[labelsField]) === -1) {
+          labels.push(item.config[labelsField]);
+        }
+        if((item.config[scaleField] || item.config[scaleField] === 0) && scales.indexOf(item.config[scaleField]) === -1) {
+          scales.push(item.config[scaleField]);
+        }
+      });
+
+
+      if(labels) {
+        labels.sort((a, b) => {
+          if(!isNaN(a) && !isNaN(b)) {
+            return a - b;
+          }
+          const valueA = parseInt(a.replace(/\D/g,''), 10);
+          const valueB = parseInt(b.replace(/\D/g,''), 10);
+          if(valueA && valueB) {
+            return valueA - valueB;
+          }
+
+          return a.localeCompare(b);
+        });
+      }
+
+      scales.sort((a, b) => a - b);
+
+      charData = {
+        labels: scales, //scales.map((s) => s.toString()),
+        datasets: labels.map((label: string, index: number) => {
+          return {
+            data: [],
+            label: label,
+            borderColor: radarDatasetColors[index].borderColor,
+            backgroundColor: radarDatasetColors[index].borderColor
+          };
+          })
+      };
+
+      labels.forEach((label: string, i: number) => {
+        scales.forEach((size: number) => {
+          const item = dataSet.benchmarks.find((b: any) => b.config[labelsField] === label && b.config[scaleField] === size);
+          if(item) {
+            charData.datasets[i].data.push(
+              { data:item.score,
+                label: size,
+                unit: chartConf.YLabel,
+                note: item.note
+              });
+          } else {
+            charData.datasets[i].data.push(null);
+          }
+        });
+      });
+    }
+
+    if(charData) {
+      if(chartConf.benchmark.includes('static_web')) {
+        (this.barChartOptionsStaticWeb as any).scales.y.title.text = chartConf.YLabel;
+        this.barChartDataStaticWeb = { labels: charData.labels, datasets: charData.datasets };
+      } else {
```

Comment:
do we enforce that it's either `static_web` or `redis`? seems fragile here, maybe do a proper check on `redis` as well (another `if` or `case`) instead of falling back to that